src/zoo/owl_zoo_cmd.ml

```diff
@@ -5,6 +5,19 @@
 
 open Owl
 
+let _read_file f =
+  let ic = open_in f in
```

Comment:
There are already `read_file` and `read_file_string` two functions in Owl_utils module. I don't see the point of making another similar function here. Please search through the code repo to make sure there are no such functions before start implementing.

Reply:
But your impl is probably more efficient than the one in Owl_utils. I can move your code there later.

Reply:
Sorry for the late reply. There I just take some familiar code snippets  to finish the task at hand, and didn't realise Owl_utils has already provided it. The efficiency is definitely not a concern here. I'm updating the code.

Reply:
No problem, I later realised that it is my fault, because the documentation does not include all the functions in Owl_utils module. I have migrated your code into `Owl_utils.read_file_string` and pushed to the master. So you can use that function now.